City-of-Helsinki/drupal-module-helfi-navigation
# UHF-8390

Guzzle::jsonDecode to utils::jsonDecode
ProphesizeTrait for prophesize method

How to test:

run `composer require drupal/helfi_navigation:dev-UHF-8390`
run `composer require drupal/upgrade_status` and `drush en -y upgrade_status`
run `drush upgrade_status:analyze helfi_navigation`
You should only see deprecation errors from prophecy / prophesize method.
